src/components/input/input.stories.mdx

```diff
@@ -86,12 +86,14 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
   </Story>
 </Canvas>
 
-## Elastic textarea
+## Elastic TextArea
 
-Add the `c-input--elastic` modifier class to a `<textarea>` element and follow the [JavaScript instructions](#javascript-instructions) to have it expand as the user types more than will fit.
+Follow the [JavaScript instructions](#javascript-instructions) below to have a `<textarea>` expand as the user types more than will fit.
 
 If the `rows` attribute is set, it will be used as the minimum height.
 
+The experience is progressively enhanced by disabling `resize` when JavaScript is enabled.
```

Comment:
Can you elaborate on why this is an enhancement? It sounds a little like "we've enhanced this element by removing a feature," which is a bit confusing.

Or maybe this is an unnecessary detail at this point?

Reply:
I'll remove this line. 👍🏽

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/2098/commits/a09fe661f395079a5ac56ba8df74d8fb63889c80